### ISSUE TITLE
Unable to resolve module `./debugger-ui/debuggerWorker.aca173c4`

### ISSUE THREAD

**Issue by dstampher** [microsoft/react-native-windows#6601]
Get Started boilerplate builds/runs, but there is this error in the npm shell that runs after executing react-native run-windows on a fresh install, following the docs.

Error: Unable to resolve module `./debugger-ui/debuggerWorker.aca173c4` from ``: ./debugger-ui/debuggerWorker.aca173c4 could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
    at ModuleResolver.resolveDependency (C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:186:15)
    at ResolutionRequest.resolveDependency (C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\node-haste\DependencyGraph.js:287:16)
    at C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\lib\transformHelpers.js:267:42
    at C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\Server.js:1096:37
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\Server.js:99:24)
    at _next (C:\Users\david\Projects\reactNativeStandard\node_modules\metro\src\Server.js:119:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

## Environment
npx react-native --version
4.13.1

npx react-native info
info Fetching system and libraries information...
System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
    Memory: 8.10 GB / 15.95 GB
  Binaries:
    Node: 12.18.4 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowAllTrustedApps: Enabled
      AllowDevelopmentWithoutDevLicense: Enabled
      Versions: 10.0.17763.0, 10.0.18362.0
  IDEs:
    Android Studio: Version  4.1.0.0 AI-201.8743.12.41.6953283
    Visual Studio: 16.8.30709.132 (Visual Studio Community 2019)
  Languages:
    Java: Not Found
    Python: 3.8.6
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.63.3 => 0.63.3
    react-native-windows: ^0.63.0-0 => 0.63.10
  npmGlobalPackages:
    *react-native*: Not Found

## Steps To Reproduce
npx react-native init reactNativeStandard --template react-native@^0.63.2
cd reactNativeStandard
npx react-native-windows-init --overwrite 
npx react-native run-windows

**Comment by NickGerleman** [comment]
This error has to do with a CLI bug recently fixed in [@react-native-community/cli 4.13.1](https://github.com/react-native-community/cli/releases/tag/v4.13.1). The error itself should be ignorable, but you should be able to avoid it by clearing your browser cache or updating your resolved CLI version to that version. Theres some more context in https://github.com/facebook/react-native/issues/28844.

**Comment by dstampher** [comment]
@NickGerleman I had an older version of react-native-cli installed globally in npm.  Deleting that, and installing the latest version of that package with yarn fixed the problem.  Thank you.

**Comment by luori-mvp77** [comment]
> This error has to do with a CLI bug recently fixed in [@react-native-community/cli 4.13.1](https://github.com/react-native-community/cli/releases/tag/v4.13.1). The error itself should be ignorable, but you should be able to avoid it by clearing your browser cache or updating your resolved CLI version to that version. Theres some more context in [facebook/react-native#28844](https://github.com/facebook/react-native/issues/28844).

I update react-native-cli to lastest version, but still find the error.
xxxxxxxx@luori-4-HVM-domU:~/_fly/workspaces/rnworkspace/secondapp$ yarn list --pattern @react-native-community/cli
yarn list v1.22.10
├─ @react-native-community/cli-debugger-ui@4.13.1
├─ @react-native-community/cli-hermes@4.13.0
├─ @react-native-community/cli-platform-android@4.13.0
├─ @react-native-community/cli-platform-ios@4.13.0
├─ @react-native-community/cli-server-api@4.13.1
├─ @react-native-community/cli-tools@4.13.0
├─ @react-native-community/cli-types@4.10.1
└─ @react-native-community/cli@4.13.1
Done in 0.88s.


None of these files exist:
  * debugger-ui/debuggerWorker.aca173c4.js(.native|.native.js|.js|.native.json|.json|.native.ts|.ts|.native.tsx|.tsx)
  * debugger-ui/debuggerWorker.aca173c4.js/index(.native|.native.js|.js|.native.json|.json|.native.ts|.ts|.native.tsx|.tsx)
    at ModuleResolver.resolveDependency (/home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:163:15)
    at ResolutionRequest.resolveDependency (/home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (/home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/node-haste/DependencyGraph.js:287:16)
    at /home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/lib/transformHelpers.js:267:42
    at Server.<anonymous> (/home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/Server.js:841:41)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/Server.js:99:24)
    at _next (/home/xxxxxx/_fly/workspaces/rnworkspace/firstapp/node_modules/metro/src/Server.js:119:9)

**Comment by chrisglein** [comment]
@luori-mvp77 This issue is closed as the original author seems unblocked. If you're still hitting this can you open a new issue that captures your whole environment information (and add a link to this one)?

**Comment by helton-isac** [comment]
I had the exact same error, 
I did all the steps and it fixed my env. I think a big difference is to **stop using npm and use yarn instead**.

 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*

My Env:
$ npx react-native --version
4.13.1

$ npx react-native info
info Fetching system and libraries information...
System:
    OS: Windows 10 10.0.19041
    CPU: (4) x64 Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz
    Memory: 1.57 GB / 7.92 GB
  Binaries:
    Node: 14.7.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - C:\Program Files\nodejs\yarn.CMD
    npm: 6.14.7 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
      AllowAllTrustedApps: Enabled
      Versions: 10.0.18362.0
  IDEs:
    Android Studio: Version  4.0.0.0 AI-193.6911.18.40.6626763
    Visual Studio: 16.7.30517.126 (Visual Studio Professional 2019)
  Languages:
    Java: Not Found
    Python: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.63.3 => 0.63.3
    react-native-windows: ^0.63.0-0 => 0.63.15
  npmGlobalPackages:
    *react-native*: Not Found
